### PR TITLE
AW-80 : Quiz

### PR DESCRIPTION
**Pull request by jasonlemay**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml [hunk 1]
```diff
@@ -13,36 +13,44 @@
...
-            </g:Button>
+        <div ui:field="quizFinished" class="{res.style.hidden} {page.style.quizEnd} {res.style.wrapper} {res.style.clearfix} {res.style.centered}">
+            <div class="{res.grid.col} {res.grid.col_1_3} {res.grid.col_push_1_3}">
+                <h2>
+                    <i class="{fonts.icons.iconArcbees}"></i>
+                    Everybody can be a bee<em>*</em>
+                </h2>
+                <p>Be the best at what you do</p>
+                <p>
+                    <em>*but you must know how to play BabyFoot table</em>
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml]: i18n for those 3 messages

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml (hunk 1 above)
Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml]: @meriouma Yes, @Pacane will be on it :)

**Pull request merged**